azure-gaming-pr/reference-architectures/cognitive-text-to-speech.md

```diff
@@ -25,20 +25,22 @@ This article will describe the architecture used in [this sample on GitHub](http
 
-There is a message limit of 245,760 bytes for each event sent.  When the result of the text to speech service is serialized, it will be too large for this buffer.  Instead, saving the result in a persistent storage and passing a pointer to the stored item as part of the return message is a feasible approach.
+When enabling this functionality in your game, keep in mind the following benefits:
 
-When enabling this functionality in your game, keep in mind the following variables:
-
-- **Voices and languages supported** - For a complete list of voices, see the [language support](https://docs.microsoft.com/azure/cognitive-services/speech-service/language-support#text-to-speech) topic.
+- **Voices and languages supported** - A large portfolio of languages and variants are supported. For a complete list of voices, see the [prebuilt neural voice](https://docs.microsoft.com/azure/cognitive-services/speech-service/language-support#prebuilt-neural-voices) and [custom neural voice](https://docs.microsoft.com/azure/cognitive-services/speech-service/language-support#custom-neural-voice). You can also specify multiple languages for Text-to-Speech output. For details, see [how to adjust speaking languages](https://docs.microsoft.com/azure/cognitive-services/speech-service/speech-synthesis-markup?tabs=csharp#adjust-speaking-languages). For custom neural voice, you can select to create [different languages](https://docs.microsoft.com/azure/cognitive-services/speech-service/language-support?tabs=speechtotext#custom-neural-voice) from single language training data.
+- **Emotional styles supported** - [Emotional tones](https://docs.microsoft.com/azure/cognitive-services/speech-service/language-support?tabs=speechtotext#voice-styles-and-roles), such as cheerful, angry, sad, excited, hopeful, friendly, unfriendly, terrified, shouting, and whispering. You can adjust the [speaking style, style degree, and role](https://docs.microsoft.com/azure/cognitive-services/speech-service/speech-synthesis-markup?tabs=csharp#adjust-speaking-styles) at the sentence level.
+- **Low latency real-time speech synthesis supported** - Use the [Speech SDK](https://docs.microsoft.com/azure/cognitive-services/speech-service/get-started-text-to-speech?tabs=terminal&pivots=programming-language-swift) or [REST API](https://docs.microsoft.com/azure/cognitive-services/speech-service/rest-text-to-speech) to convert text-to-speech by using prebuilt neural voices or custom neural voices.
+- **Visemes supported** - You can use [visemes](https://docs.microsoft.com/azure/cognitive-services/speech-service/how-to-speech-synthesis-viseme?pivots=programming-language-csharp) during real-time synthesizing to control the movement of 2D and 3D avatar models, so that the mouth movements are perfectly matched to synthetic speech. 
```

Comment:
update the link to exclude the pivot.

https://docs.microsoft.com/azure/cognitive-services/speech-service/how-to-speech-synthesis-viseme

Reply:
Updated. Thanks.